website/app.py

```diff
@@ -128,9 +126,6 @@ def init_app(settings_module='website.settings', set_backends=True, routes=True,
         sentry.init_app(app)
         logger.info("Sentry enabled; Flask's debug mode disabled")
 
-    if set_backends and fixtures:
```

Comment:
Now that `fixtures` is unused, it can be removed from the function signature. Be sure to remove any usages of that param, e.g. in manage.py.